Game/Kasia robocze/css2.css

```diff
@@ -0,0 +1,1520 @@
+  --color-grey: #887575a6;
+}
+
+/*smooth scroll*/
+
+html {
+  scroll-behavior: smooth;
+}
+
+/**navigation toolbar**/
+
+.site-header {
+  align-items: stretch;
+  max-width: 50%;
+  flex-grow: 1;
+  position: fixed;
+  z-index: 1;
+  height: 100%;
+  font-family: var(--font-title);
+  z-index: 1;
+  position: fixed;
+  left: 0;
+}
+
+.site-logo {
+  max-height: 150px;
+  max-width: 400px;
+  float: right;
+}
+
+.site-logo .logo {
+  max-height: 100px;
+  height: 100%;
+  align-content: center;
+  margin-top: var(--gutter);
+  color: white;
+}
+
+.site-logo .name {
+  max-height: 100px;
+  height: 100%;
+  align-content: center;
+  margin-top: var(--gutter);
+}
+
+@media (max-width: 500px) {
+  .site-logo .name {
+    display: none;
+  }
+}
+
+.icon-1, .icon-2, .icon-3 {
+  position: absolute;
+  top: 5%vw;
+  width: 32px;
+  height: 3px;
+  background-color: white;
+  transition: all 400ms cubic-bezier(.84, .06, .52, 1.8);
+  box-shadow: var(--color-dark-grey) 2px 1px 4px;
+}
+
+.icon-1 {
+  transform: translateY(-8px);
+  animation-delay: 100ms;
+}
+
+.icon-3 {
+  transform: translateY(8px);
+  animation-delay: 250ms;
+}
+
+.hamburger-icon {
+  position: absolute;
+  height: 60px;
+  width: 60px;
+  top: calc(var(--gutter)*3.5);
+  left: calc(var(--gutter)*2);
+  z-index: 1000;
+  cursor: pointer;
+  transition: all 0.2s ease-in-out;
+}
+
+.icon-1.a {
+  transform: rotate(40deg);
+}
+
+.icon-3.b {
+  transform: rotate(-40deg);
+}
+
+.icon-2.c {
+  opacity: 0;
+}
+
+.clear {
+  clear: both;
+}
+
+.main-menu {
+  display: none;
+  position: absolute;
+  top: 15vh;
+  height: 100%;
+  /*transition: all 600ms cubic-bezier(.62, .04, .3, 1.56);
+  transition-delay: 100ms;*/
+  list-style: none;
+  width: 100%;
+}
+
+.main-menu-item a {
+  text-overflow: initial;
+  color: white;
+  text-decoration: none;
+  float: left;
+  transition: border-top-color 0.2s;
+  margin-left: 20px;
+  font-size: 1.5rem;
+  text-transform: uppercase;
+  padding-top: 2rem;
+  margin-top: 0;
+  min-width: 200px;
+}
+
+.main-menu-item a:hover {
+  font-weight: 1000;
+  font-size: 2rem;
+}
+
+@media (max-width: 1000px) {
+  .site-header .main-menu-item a {
+    font-size: 18px;
+  }
+  .site-logo .logo {
+    max-height: 90px;
+  }
+  .site-logo .name {
+    max-height: 90px;
+  }
+   .menu-curtain.move {
+    min-width: 130px;
+}
+}
+@media (max-width: 800px) {
+  .site-header .main-menu-item a {
+    font-size: 15px;
+  }
+  .site-logo .logo {
+    max-height: 80px;
+  }
+  .site-logo .name {
+    max-height: 80px;
+  }
+  .icon-1, .icon-2, .icon-3 {
+    height: 3px;
+    width: 18px;
+  }
+  .menu-curtain.move {
+    min-width: 100px;
+  }
+}
+
+.menu-curtain {
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: rgba(67, 66, 71, 0.75);
+  height: 100%;
+  width: 0%;
+  transition: all 500ms cubic-bezier(.62, .04, .3, 1.8);
+  transition-delay: 50ms;
+  
+  opacity: 1;
+}
+
+/**wysuwane menu**/
+
+.menu-curtain.move {
+  width: 18vw;
+  min-width: 150px;
```

Comment:
To z powodu tego min-width tak skacze, jak usuniesz, to jest ok. I popracowałabym nad tymi szerkościami, bo przy niektórych tło jest za wąskie w stosunku do długości wyrazów, szczególnie jak się na nie najedzie i np zlewają się z białym tłem jak na nie trafią.